.vscode/settings.json

```diff
@@ -3,6 +3,17 @@
   "editor.codeActionsOnSave": {
     "source.organizeImports": true
   },
-  "python.analysis.extraPaths": ["./generated"],
-  "python.formatting.provider": "black"
-}
+  "editor.rulers": [
```

Comment:
Looks like you accidentally committed this!

Reply:
this was intentional - I was just adding the stuff we have in zuuul that we needed here to
1. make it easier to keep code linted
2. make black behave rationally

Reply:
...at least on my machine. the rulers are helpful to make the linter happy and black does wild things with the imports if it is allowed to touch them.